test/e2e/tests/add-hide-token.spec.js

```diff
@@ -69,12 +95,11 @@ describe('Add existing token using search', function () {
   it('renders the balance for the chosen token', async function () {
     await withFixtures(
       {
-        fixtures: 'imported-account',
+        fixtures: new FixtureBuilder()
+          .withPreferencesController({ useTokenDetection: true })
```

Comment:
was wondering whether we need `useTokenDetection` here?

Reply:
Good question.
The current imported account fixture on develop has token detection switched on. See [here](https://github.com/MetaMask/metamask-extension/blob/65e3047a748d2a9b428959801d098831d1d7c532/test/e2e/fixtures/imported-account/state.json#L149) 
Initially I had token detection off, so had to add the token to state. See [here](https://github.com/MetaMask/metamask-extension/pull/16061#discussion_r993968925)
But after discussion with Mark above, I realised it's better to remove this extra state and enable token detection to bring the test inline with how it was before.
So with token detection switched on, the extension will now call the token api which is mocked. See [here](https://github.com/MetaMask/metamask-extension/blob/65e3047a748d2a9b428959801d098831d1d7c532/test/e2e/mock-e2e.js#L153)